lib/osf-components/addon/components/cedar-metadata-editor/component.ts

```diff
@@ -0,0 +1,23 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+// import { task } from 'ember-concurrency';
+import { tracked } from 'tracked-built-ins';
+import config from 'ember-osf-web/config/environment';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const { cedarConfig } = config;
+
+interface Args {
+    instanceObj: any;
+    templateOjb: any;
+}
+
+export default class CedarMetadataEditor extends Component<Args> {
```

Comment:
This file has `this.args.instanceObj` as well as `this.instanceObj`. We should probably only have one of these for readability, or define `this.instanceObj = this.args.instanceObj` for brevity.